UNRELEASED.md

```diff
@@ -10,6 +10,7 @@ Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to f
 
 - Added optional `videoProgress` and `showVideoProgress` props to `VideoThumbnail` for video progress indicator ([#3057](https://github.com/Shopify/polaris-react/pull/3057))
 - Enabled much easier tree-shaking in consuming apps by having a multi-file build instead of a single-file build ([#3137](https://github.com/Shopify/polaris-react/pull/3137))
+- Added optional `isFiltered` prop to `ResourceList` to conditionally render more informative select all button label ([#3153](https://github.com/Shopify/polaris-react/pull/3153))
```

Comment:
Looking at the custom fork we have in Products (`ResourceListWithHeader`). It looks like it's already supporting this feature with the `filtersApplied` prop. I feel we should match the prop's name to make it consistent. 
What do you think?

Reply:
i went with this name because `filtersApplied` doesn't seem like a boolean to me, but rather, an array of applied filters. but i'm not strongly opinionated, we can change it if you believe it's more accurate!

Reply:
Let's let Polaris folks decide based on the conventions they try to enforce in this repo.
cc/ @alex-page

Reply:
Hey @loic-d I would agree with @carolopolo. From my understanding we use `isX` a few times in Polaris.
https://github.com/Shopify/polaris-react/blob/master/src/components/Avatar/Avatar.tsx#L49